### DISCUSSION TITLE
Sidebar examples

### DISCUSSION THREAD

**Discussion by jjc54** [quarto-dev/quarto-cli#9182] [Q&A]
### What would you like to do?

Give feedback or suggest an improvement

### Description

I'm having a difficult time finding examples on adding sidebar content without using Shiny for Dashboards 1.4!

**Comment by mcanouil** [comment]
Could you elaborate? What is the context exactly?
Did you look at the search results within Quarto website for "sidebar"?

**Comment by jjc54** [comment]
Yes! The only examples of "sidebar" that I've found require shiny/servers. It was inferred through the Dashboard documentation that it was possible to implement sidebars without shiny, but all the dashboard examples empty sidebars or "Sidebar content." Placeholders. Unless I'm misunderstanding and dashboard sidebar requires Shiny!

**Comment by mcanouil** [comment]
What's your goal?

What else do you want besides https://quarto.org/docs/dashboards/inputs.html#global-sidebar

The example shows that you can put any arbitrary content.

A sidebar is only a level one or two header with `{.sidebar}`.
Then the content is 100% up to you.

**Comment by jjc54** [comment]
To add a card input to one of my plotly figures that allows me to filter by one of my variables
